src/nomad_simulations/schema_packages/numerical_settings.py

```diff
@@ -123,25 +109,114 @@ class Mesh(ArchiveSection):
+
+
+class NumericalIntegration(NumericalSettings):
+    """
+    Numerical integration settings used to resolve the following type of integrals by discrete
+    numerical integration:
+
+    ```math
+    \int_{\vec{r}_a}^{\vec{r}_b} d^3 \vec{r} F(\vec{r}) \approx \sum_{n=a}^{b} w(\vec{r}_n) F(\vec{r}_n)
+    ```
+
+    Here, $F$ can be any type of function which would define the type of rules that can be applied
+    to solve such integral (e.g., 1D Gaussian quadrature rule or multi-dimensional `angular` rules like the
+    Lebedev quadrature rule).
+
+    These multidimensional integral has a `Mesh` defined over which the integration is performed, i.e., the
+    $\vec{r}_n$ points.
+    """
+
+    mesh = SubSection(sub_section=Mesh.m_def)
+
+    coordinate = Quantity(
+        type=MEnum('full', 'radial', 'angular'),
+        description="""
+        Coordinate over which the integration is performed. `full` means the integration is performed in
+        entire space. `radial` and `angular` describe cases where the integration is performed for
+        functions which can be splitted into radial and angular distributions (e.g., orbital wavefunctions).
+        """,
+    )
+
+    integration_rule = Quantity(
+        type=MEnum(
+            # radial Gaussian rules
+            'Gauss-Legendre',  # 1-D
+            'Gauss-Chebyshev',
+            'Gauss-Lobatto',
+            'Gauss-Hermite',
+            'Gauss-Laguerre',  # for STO/Slater codes
+            'Euler-Maclaurin',  # Murray–Handy–Laming ’93
+            'Logarithmic-Log3',  # Mura–Knowles ’96
+            'Treutler-Ahlrichs',  # Turbomole M4 mapping
+            'Becke-Radial',  # Gauss-Chebyshev (Becke ’88)
+            'MultiExp',  # Chien-Gill ’06 (SG-0 parent)
+            'Double-Exponential',  # tanh-sinh (SG-2 / SG-3)
+            # angular grids
+            'Lebedev',  # any (N,θ,φ) order
+            'Lebedev-Laikov',  # re-optimisation by Laikov enabling very high orders
+            'Product-Legendre',  # Gauss-Legendre θ × uniform φ
+            'Becke',  # Becke multicentre partitioning
+            'SG-1',  # Gill–Head-Gordon Smolyak grids
+            'SG-2',
+            'SG-3',
+        ),
+        description="""
+        Integration rule used. This can be any 1D Gaussian quadrature rule or multi-dimensional `angular` rules,
+        e.g., Lebedev quadrature rule (see e.g., Becke, Chem. Phys. 88, 2547 (1988)).
+        """,
+    )
+
+    integration_thresh = Quantity(
         type=np.float64,
-        shape=['n_points'],
         description="""
-        Weight of each point. A value smaller than 1, typically indicates a symmetry operation that was
-        applied to the mesh. This quantity is equivalent to `multiplicities`:
+        An accuracy threshold for the integration grid, controlling how fine the 
+        discretization is. Some programs label it "integral accuracy" or "grid accuracy".
+        For instance, GRIDTHR in Molpro or BFCut in ORCA.
```

Comment:
I think that this line may be removed once the parser mapping has been implemented.
Your explanation on the previous line is more approachable to other communtities and suffices.